### DISCUSSION TITLE
Version of pdf file encoding

### DISCUSSION THREAD

**Discussion by glennvanderveer** [TomRoush/PdfBox-Android#332] [Q&A]
Is there a maximum version of pdf encoding that the pdf file should be encoded with?  I have files that are encoded with version 1.4 (Acrobat 5.x) that I can display, and files that are encoded with version 1.7 (Acrobat 8.x) that cause the .renderImage call to crash.

**Comment by TomRoush** [comment]
1.7 shouldn't be a problem, so it sounds like there's something in else that PDF that causing the crashing. If you are able to share the PDF, we can create and issue and take a look at the problem
